compat/v0/compat.go

```diff
@@ -532,6 +535,19 @@ func AddressEncodeDecoder(chain multichain.Chain, network multichain.Network) mu
 	case multichain.Zcash:
 		params := ZcashNetParams(network)
 		return zcash.NewAddressEncodeDecoder(params)
+	case multichain.Arbitrum,
+		multichain.Avalanche,
+		multichain.BinanceSmartChain,
+		multichain.Ethereum,
+		multichain.Goerli,
+		multichain.Moonbeam:
+		return ethereum.NewAddressEncodeDecoder()
+	case multichain.Fantom:
+		return fantom.NewAddressEncodeDecoder()
+	case multichain.Polygon:
+		return polygon.NewAddressEncodeDecoder()
+	case multichain.Solana:
+		return solana.NewAddressEncodeDecoder()
```

Comment:
Should we move this function to another package? These chains aren't relevant for the v0 compat.